src/stable/lib/execute_with_candid_serde.ts

```diff
@@ -120,9 +120,20 @@ async function getUnencodedResult(
         );
 
         if (result === true) {
+            if (
+                globalThis._azleIcStable === undefined &&
+                globalThis._azleIcExperimental === undefined
+            ) {
+                throw new Error(
```

Comment:
Shouldn't this be after the two ifs to follow the pattern in all of the other files?